draft/src/api.rs

```diff
@@ -0,0 +1,28 @@
+#[derive(Debug)]
+#[derive(Clone)]
+pub struct Card {
+    pub name: String,
+}
+
+impl Card {
+    pub fn new(name: &String) -> Self {
```

Comment:
&str accepts &String and others so is preferred.

Reply:
Done.